metrics/identical_codeblocks.py

```diff
@@ -8,6 +8,15 @@
 
 TOKENS = 35
 
+_CHARDET_ENCODINGS_TO_CPD = {
+    "UTF-8-SIG": "UTF-8",
+    "utf-8": "UTF-8",
+    "UTF-16BE": "UTF-16BE",
+    "UTF-16LE": "UTF-16LE",
+    "ascii": "US-ASCII",
+    "ISO-8859-1": "ISO-8859-1",
+}
```

Comment:
Would love feedback as to if this dictionary looks correct. It should translate the output of chardet to the corresponding charset name of CPD.

Reply:
same as my comment to the last one. Furthermore, the CPD documentation is quite limited in terms of the types of encodings as it just refers to the 'standard charsets'.

Reply:
I looked into the documentation and added a page about it in notion under technical notes